ocaml/xenopsd/xc/device.ml

```diff
@@ -3696,7 +3698,8 @@ module Dm = struct
       | Restore ->
           qemu_args ~xs ~dm info true domid
     in
-    debug "Device.Dm.start domid=%d args: [%s]" domid
+    debug "Device.Dm.start domid=%d restore=%s qemu args: [%s]" domid
```

Comment:
This would print `restore=Start qemu`. I think we don't want `restore=`.

Reply:
I'll change it to print `Device.Dm.start domid=XX action=Start qemu args:`. It's not ideal because of the repetition, but should be used for debugging only, so it makes sense to look for action there